java/runtime/src/main/java/org/ray/runtime/object/ObjectSerializer.java

```diff
@@ -21,11 +22,13 @@
       .valueOf(ErrorType.ACTOR_DIED.getNumber()).getBytes();
   private static final byte[] UNRECONSTRUCTABLE_EXCEPTION_META = String
       .valueOf(ErrorType.OBJECT_UNRECONSTRUCTABLE.getNumber()).getBytes();
-
   private static final byte[] TASK_EXECUTION_EXCEPTION_META = String
       .valueOf(ErrorType.TASK_EXECUTION_EXCEPTION.getNumber()).getBytes();
 
-  private static final byte[] RAW_TYPE_META = "RAW".getBytes();
+  public static final byte[] OBJECT_METADATA_TYPE_CROSS_LANGUAGE = "Cross".getBytes();
```

Comment:
I will refine the metadata formats in future PR. `"Cross"` is defined in design doc, `"CROSS_LANGUAGE"` will be clearer, but this will use more bytes.

Reply:
using more bytes doesn't look like a big deal, there are more overheads elsewhere though. I think readability is more important. But if you really care about the length, maybe `XLANG`?

Reply:
`XLANG` is clearer than `Cross`, and I will rename all the metadata bytes to upper case.